core/src/main/java/com/linecorp/armeria/client/Http1ResponseDecoder.java

```diff
@@ -174,16 +165,18 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                         assert res != null;
                         this.res = res;
 
-                        res.logResponseFirstBytesTransferred();
-
+                        res.startResponse();
+                        final ResponseHeaders responseHeaders = ArmeriaHttpUtil.toArmeria(nettyRes);
+                        final boolean written;
                         if (nettyRes.status().codeClass() == HttpStatusClass.INFORMATIONAL) {
                             state = State.NEED_INFORMATIONAL_DATA;
+                            written = res.tryWrite(responseHeaders);
```

Comment:
What do you think of adding `res.writeInformationalHeaders()` as well for consistency?

Reply:
I think it's not a good idea because:
- `res.writeInformationalHeaders()` just delegates to `delegate.tryWrite(o);`
- I have to call `tryWriteResponseHeaders` even if the status is 1XX for WebSocket upgrade response.

Reply:
If we mix to use `tryWrite()` and `tryWriteXXX()`, after some time, I think we will easily get confused about which function to use.

> I have to call tryWriteResponseHeaders even if the status is 1XX for WebSocket upgrade response.

Q: If so, the following condition is no longer valid?
https://github.com/line/armeria/blob/main/core/src/main/java/com/linecorp/armeria/client/HttpResponseDecoder.java#L263-L263

Reply:
> If so, the following condition is no longer valid?

That is correct. I was going to remove it when I implement WebSocketClient. 😉